schemas/answers/unit.json

```diff
@@ -85,7 +91,8 @@
       "id",
       "type",
       "mandatory",
-      "unit"
+      "length",
+      "use_plural_for_label"
```

Comment:
I think these two could be `unit_length` & `unit_plural` as they are both related to unit, but not bothered either way 👍